qa-core/src/config/internal-api/fixtures/table.ts

```diff
@@ -0,0 +1,60 @@
+import { Row } from "@budibase/types"
+
+type Table = {
```

Comment:
We can import the existing definition for a table from budibase. If there is data missing from that definition we should add to the existing one so we aren't repeating ourselves

Reply:
Done